module/documents/actor/actor.mjs

```diff
@@ -1779,18 +1779,30 @@ export default class Actor5e extends Actor {
    * carried by an Actor.
    * @returns {Promise<Actor5e>}
    */
+
   convertCurrency() {
     const curr = foundry.utils.deepClone(this.system.currency);
```

Comment:
It would probably be a good time to change this to the whole word for clarity.

```suggestion
    const currency = foundry.utils.deepClone(this.system.currency);
```